INSTALL

```diff
@@ -540,8 +540,9 @@ The specific libraries and their options are:
   See also the "Specific options for hypre" section at the end of this file.
   URL: https://github.com/hypre-space/hypre and https://www.llnl.gov/casc/hypre
   Options: HYPRE_OPT, HYPRE_LIB.
-  Versions: HYPRE >= 2.10.0b,
-            HYPRE >= 2.20.0 for '--enable-mixedint' support.
+  Versions: HYPRE >= 2.10.0b  (HYPRE built without CUDA)
+            HYPRE >= 2.20.0   (for '--enable-mixedint' support.)
+            HYPRE >= 2.22.0   (HYPRE built with CUDA)
```

Comment:
```suggestion
  Versions: HYPRE >= 2.22.0   (HYPRE built with CUDA or '--enable-mixedint')
            HYPRE >= 2.10.0b  (default HYPRE build)
```

Reply:
I like the current text better.

Reply:
The new one is more clear to me, but I don't feel strongly about it.

Can we rephrase `(for '--enable-mixedint' support.)` to `(HYPRE built with '--enable-mixedint')`, or at least drop the period?

Reply:
> The new one is more clear to me, but I don't feel strongly about it.

Note that the versions for mixedint and cuda are different: 2.20.0 vs 2.22.0.

> Can we rephrase `(for '--enable-mixedint' support.)` to `(HYPRE built with '--enable-mixedint')`, or at least drop the period?

Sounds good to me.